src/app/components/file-uploader/file-uploader.component.ts

```diff
@@ -0,0 +1,157 @@
+import { Component, Input, ViewChild, ElementRef, OnInit } from '@angular/core';
+import { UploadFile } from 'src/app/models/domain/uploadFile';
+import { FileUploaderService } from 'src/app/services/file-uploader.service';
+import { map } from 'rxjs/operators';
+import { HttpEventType } from '@angular/common/http';
+import { forkJoin, Observable, of } from 'rxjs';
+import { AlertConfig } from '../../models/internal/alert-config';
+import { AlertType } from '../../models/internal/alert-type';
+import { AlertService } from '../../services/alert.service';
+
+@Component({
```

Comment:
Component documentation is missing, what is the functionality/purpose of this component?